README.md

```diff
@@ -257,7 +257,7 @@ Follow the instructions below to set up the remote write:
 kubectl create secret generic aws-prometheus-credentials --from-literal AccessKey=${ACCESS_KEY} --from-literal SecretAccessKey=${SECRET_ACCESS_KEY}
 ```
 
-2. Copy the [prometheus-remote-write-aws.yaml](./monitoring/prometheus-remote-write-aws.yaml) file and replace the values of the these variables:
+2. Copy the [prometheus-remote-write-aws.yaml](https://github.com/risingwavelabs/risingwave-operator/blob/main/monitoring/kube-prometheus-stack/prometheus-remote-write-aws.yaml) file and replace the values of the these variables:
```

Comment:
It would be better to use the relative path, so the link works in branches other than the main.

```suggestion
2. Copy the [prometheus-remote-write-aws.yaml](./monitoring/kube-prometheus-stack/prometheus-remote-write-aws.yaml) file and replace the values of the these variables:
```

Reply:
Yes, I think so